logger/fleet_commander_logger.js

```diff
@@ -642,6 +644,271 @@ GSettingsLogger.prototype._bus_name_disappeared_cb = function (connection, bus_n
     this.dconf_subscription_id = 0;
 }
 
+
+var ChromiumLogger = function (connmgr, datadir, namespace, interval, bookmarks_interval) {
+    this.datadir = datadir || GLib.getenv('HOME') + '/.config/chromium';
```

Comment:
I'm nitpicking here, but isn't it better adding the default value in the arguments?

Reply:
Ah, this is JavaScript, nevermind!

Reply:
This is madness!
Madness?
This ... is ... JAVASCRIPT!